index.html

```diff
@@ -17,27 +17,196 @@
+                <li class="class-project-li">HTML/CSS</li>
+                <li class="class-project-li">Ruby on rails</li>
+                <li class="class-project-li">Javascript</li>
+              </ul>
+              <a href="#" target="_parent" class="class-project-link">
+                <span class="class-project-button">See this Project</span>
+                <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+              </a>
+            </div>
+          </li>
+        </ul>
+      </section>
+
+      <!--The section about my self-->
+      <section id="id-section-about-my-self">
+
+        <!--Introduction to the the project list-->
+        <h1 class="class-sections-titles">About</h1>
+
+        <!--litle description about me-->
+        <p class="class-about-description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
+
+        <!--Button to the CV-->
+        <h2 class="class-about-cv">
+          <span class="class-about-span">Get my CV</span>
+          <img src="./images/ic_download.svg" alt="union pic" class="class-about-pic">
+        </h2>
+
+        <!--Languages used-->
+        <div class="class-about-div" id="id-about-languages">
+          <h2 class="class-about-div-title">Languages</h2>
+          <ul class="class-about-div-ul">
+            <li class="class-div-list"><span>JavaScript(ES5/ES6)</span></li>
+            <li class="class-div-list"><span>Ruby</span></li>
+            <li class="class-div-list"><span>HTML</span></li>
+            <li class="class-div-list"><span>CSS</span></li>
+          </ul>
+        </div>
+
+        <!--Frameworks used-->
+        <div class="class-about-div" id="id-about-frameworks">
+          <h2 class="class-about-div-title">Frameworks</h2>        
+          <ul class="class-about-div-ul">
+            <li class="class-div-list"><span>Bootstrap</span></li>
+            <li class="class-div-list"><span>Ruby on rails</span></li>
+            <li class="class-div-list"><span>RSpec</span></li>
+            <li class="class-div-list"><span>Capibara</span></li>
+            <li class="class-div-list"><span>Selenium</span></li>
+          </ul>
+        </div>
+
+        <!--All skills-->
+        <div class="class-about-div" id="id-about-skills">          
+          <h2 class="class-about-div-title">Skills</h2>
+          <ul class="class-about-div-ul">
+            <li class="class-div-list"><span>Codekit</span></li>
+            <li class="class-div-list"><span>Codepen</span></li>
+            <li class="class-div-list"><span>GitHub</span></li>
+            <li class="class-div-list"><span>GitLAb</span></li>
+            <li class="class-div-list"><span>Terminal</span></li>
+          </ul>
+        </div>
```

Comment:
- [ ] _[Optional]_ The technology categories ("Languages", "Frameworks" and "Skills") is a list itself and should be grouped with the `ul` tag, and each of them in a `li` just as the first reviewer pointed out although 

>you have grouped the technology in each category ("JavaScript", "HTML" and "CSS") using the `ul` tag and each of them a `li`.

See an illustration below:
```html
<ul class="categories">
    <li>
        <p>Languages</p>
        <ul class="technology">
            <li>JavaScript</li>
            <li>Ruby</li>
            <li>HTML</li>
        </ul>
    </li>
    <li>
        <p>Languages</p>
        <ul class="technology">
            <li>JavaScript</li>
            <li>Ruby</li>
            <li>HTML</li>
        </ul>
    </li>
    <li>
        <p>Languages</p>
        <ul class="technology">
            <li>JavaScript</li>
            <li>Ruby</li>
            <li>HTML</li>
        </ul>
    </li>
</ul>
```
-----
_Try to make the changes before you merge_

___I only marked this `Optional` because it's your first week. Next time, you might have to repeat the week if your project doesn't fully meet the requirements. Henceforth, please make sure you make all the mandatory changes required by a Code Reviewer.___